.github/workflows/ci.yaml

```diff
@@ -187,6 +187,7 @@ jobs:
         env:
           RN_UI_PATH: react-native-ui
           NODE_UI_PATH: node-ui
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
```

Comment:
We need this for the recording for branches originating from the original repo `ory/kratos`, so I am leaving it in here.